pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -245,34 +245,32 @@ func (qt *QueueTracker) setLimit(queuePath string, maxResource *resources.Resour
 	childQueueTracker.maxResources = maxResource
 }
 
-func (qt *QueueTracker) headroom(queuePath string) *resources.Resource {
+func (qt *QueueTracker) headroom(hierarchy []string) *resources.Resource {
```

Comment:
I suggest receiving the full path and using `getChildQueuePath()` from `utilities.go` to keep the code consistent (also the UGM API remains the same and simpler).

Reply:
That utility function will be removed as part of a next change